client.go

```diff
@@ -0,0 +1,179 @@
+package luasandbox
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"reflect"
+	"time"
+
+	"github.com/calyptia/api/types"
+)
+
+type Client struct {
+	url        string
+	httpClient http.Client
+	nextId     int
+}
+
+type params struct {
+	Events []types.FluentBitLogAttrs `json:"events"`
+	Filter string                    `json:"filter"`
+}
+
+type request struct {
+	JsonRpcVersion string `json:"jsonrpc"`
+	Id             int    `json:"id"`
```

Comment:
```suggestion
	ID             int    `json:"id"`
```

Is mot idiomatic this way.